opentelemetry/src/sdk/trace/provider.rs

```diff
@@ -116,11 +116,11 @@ impl Builder {
         // drop. We cannot assume we are in a multi-threaded tokio runtime here, so use
         // `spawn_blocking` to avoid blocking the main thread.
         let spawn = |fut| tokio::task::spawn_blocking(|| futures::executor::block_on(fut));
-        let batch = sdk::trace::BatchSpanProcessor::builder(
+        let batch = sdk::trace::BatchSpanProcessor::from_env(
             exporter,
             spawn,
-            tokio::time::sleep,
             crate::util::tokio_interval_stream,
+            tokio::time::sleep,
```

Comment:
Oddly, from_env switches these two last arguments :(

Reply:
yeah that looks like an oversight, want to create a PR to fix that?